modules/percolator/src/main/java/org/elasticsearch/percolator/PercolatorHighlightSubFetchPhase.java

```diff
@@ -40,6 +40,16 @@
         this.highlightPhase = new HighlightPhase(highlighters);
     }
 
+    @Override
+    public String name() {
+        return "percolator_highlight";
+    }
```

Comment:
These power the `type` and `description` fields on the `children` field in the fetch profile.